deployment/tests/f29lite.hosts

```diff
@@ -0,0 +1,9 @@
+buddhi ansible_host=<buddhi-host> ansible_user=<buddhi-user>
```

Comment:
this file is for your own tests. Not for everyone.

Reply:
Yes that's right. It's been removed along with some others.